### ISSUE TITLE
number of dataloader

### ISSUE THREAD

**Issue by sc-56** [xu-ji/IIC#83]
Sorry for bothering,

Recently, I change the number of dataloader to train the MNIST. I notice if I reduce the number of dataloader, the accuracy increasing rate seems to be slower than before. I'm curious whether the result is able to achieve oustanding performance with longer time when I use 1 dataloader, other than 5?

Thank you so much for your support.

**Comment by xu-ji** [comment]
Talking about number of dataloaders in the paper:

> For data augmentation, we repeat images within each batch r times; this means that multiple image pairs within a batch contain the same original image, each paired with a different transformation, which encourages greater distillation since there are more examples of which visual details to ignore

So, it makes sense that more dataloaders improves performance. I do remember that this benefit did not seem to be replaceable by using 1 dataloader with longer training time, probably because the quality of each training signal per training timestep is higher with n>1 dataloaders (i.e. it's better to be able to see n versions of the same image in one batch/one epoch when using n dataloaders, rather than having to wait 5 epochs to see 5 versions of the same image with 1 dataloader).

But the model should still learning something meaningful with 1 dataloader, just not as well.

**Comment by sc-56** [comment]
Thank you so much for your detailed explanation.
